rubocop.yml

```diff
@@ -50,6 +49,12 @@ Style/AccessorMethodName:
   Enabled: false
 Style/Alias:
   Enabled: false
+Style/BlockDelimiters:
+  EnforcedStyle: semantic
```

Comment:
Thanks. Looking at them,  I consider these pretty much spot on, and would change as advised by rubocop. If we go with simply linecount, then all `expect { block }.to change { another_block }` will explode (where either block has more than one line).

One thing you can do here is [edit the list](http://rubocop.readthedocs.io/en/latest/cops_style/#styleblockdelimiters) of procedural and functional methods to adjust to your usage - and either add them in a PR here, on in Kiev's own rubocop.yml which inherits from this. It's okay to have separate rules on other projects, as every one is different.